kafka/ci/resources/kafka.yaml

```diff
@@ -305,8 +305,8 @@ init_config:
         bean: 'kafka.server:type=KafkaRequestHandlerPool,name=RequestHandlerAvgIdlePercent'
         attribute:
           Count:
-            metric_type: rate
-            alias: kafka.request.handler.avg.idle.pct.rate
+            metric_type: gauge
```

Comment:
I think I agree with the notion that this should probably have been a gauge from day one. But this change, at this stage, would bring in a backward incompatibility problem that could break graphs and/or monitors. We could maybe look into greater depth into that (ie. gauging how many customers would be impacted).... But alternatively we could just ship the metric in the two forms, `rate` and `gauge` with the different names...